pygments/lexers/c_like.py

```diff
@@ -663,3 +649,56 @@ class OmgIdlLexer(CLexer):
             include('annotation_appl'),
         ],
     }
+class FlexLexer(CLexer):
+    """
+    Lexer for Flex <http://dinosaur.compilertools.net/flex/index.html>, 
```

Comment:
```suggestion
    Lexer for `Flex <http://dinosaur.compilertools.net/flex/index.html>`_, 
```

to make it a real hyperlink.

Reply:
Actually the hyperlink should go to the `url` attribute of the class. `https://github.com/westes/flex` is probably a better choice, since it's not from 1995 :)